src/ui/camera.js

```diff
@@ -317,6 +317,13 @@ class Camera extends Evented {
      * });
      */
     zoomTo(zoom: number, options: ? AnimationOptions, eventData?: Object): this {
+        if (zoom !== number){
```

Comment:
I don't think this will work (number is a flow annotation, not a variable.) Maybe something like the following will provide the intended behavior?
```suggestion
        if (typeof zoom !== 'number'){
```